src/controllers/users.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const usersServices = require('../services/users');
+
+function getAll(req, res) {
+  const users = usersServices.getAll();
+
+  res.statusCode = 200;
+  res.send(users);
+}
+
+function getById(req, res) {
+  const { userId } = req.params;
+
+  if (!userId) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const foundUser = usersServices.getById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.statusCode = 200;
+  res.send(foundUser);
+}
+
+function add(req, res) {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = usersServices.create(name);
+
+  res.statusCode = 201;
+  res.send(newUser);
+}
+
+function remove(req, res) {
+  const { userId } = req.params;
+
+  const userToRemove = usersServices.getById(userId);
+
+  if (!userToRemove) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  usersServices.remove(userId);
+  res.sendStatus(204);
+}
+
+function update(req, res) {
+  const { userId } = req.params;
+  const { name } = req.body;
+
+  if (!userId || !name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
```

Comment:
Combine similar logic
```suggestion
  if (!userId || !name || typeof name !== 'string') {
    res.sendStatus(400);

    return;
  }
```